### DISCUSSION TITLE
What is the best practise configs for Next.js + Tailwind?

### DISCUSSION THREAD

**Discussion by rikusen0335** [vercel/next.js#20964] [Help]
Nowadays, we have lots of documents and article, but I don't sure which one is exact one to use.
Because of, the frameworks for JavaScript/TypeScript are changing really fast so I wanna know currently great example.

What is your idea?

**Comment by michrome** [comment]
@rikusen0335 Tailwind's [Install Tailwind CSS with Next.js](https://tailwindcss.com/docs/guides/nextjs) article is clear, correct and current for Tailwind 2 and Next 10.
